examples/src/main/java/io/dapr/springboot/OpenTelemetryConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import io.dapr.examples.invoke.http.InvokeClient;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.exporters.logging.LoggingSpanExporter;
+import io.opentelemetry.exporters.zipkin.ZipkinSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Tracer;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class})
+public class OpenTelemetryConfig {

```

Comment:
Do we want to integrate with Dapr configuration?

Reply:
I am not sure if I understand the proposal here. This is an example of how users can integrate with OpenTelemetry, but Dapr should not dictate that - this is 100% up to the application to decide how to initialize OpenTelemetry in their app.